instrumentation/opentelemetry-instrumentation-aws-lambda/src/opentelemetry/instrumentation/aws_lambda/__init__.py

```diff
@@ -189,8 +189,14 @@ def _instrumented_lambda_handler_call(
 
         # See more:
         # https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html
+        # https://docs.aws.amazon.com/lambda/latest/dg/with-sns.html
+        # https://docs.aws.amazon.com/AmazonS3/latest/userguide/notification-content-structure.html
+        # https://docs.aws.amazon.com/lambda/latest/dg/with-ddb.html
+        span_kind = SpanKind.SERVER
```

Comment:
Let me know what you think of these suggestions. We definitely want to fix all code branches even the default branch, but I like using the code to explain that.

```suggestion
span_kind = None
```